src/30_Advanced/Embedding_videos_in_amp-carousel.html

```diff
@@ -0,0 +1,140 @@
+  <script async custom-element="amp-instagram" src="https://cdn.ampproject.org/v0/amp-instagram-0.1.js"></script>
+  <script async custom-element="amp-vine" src="https://cdn.ampproject.org/v0/amp-vine-0.1.js"></script>
+  <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
+  <script async custom-element="amp-vimeo" src="https://cdn.ampproject.org/v0/amp-vimeo-0.1.js"></script>
+  <script async custom-element="amp-dailymotion" src="https://cdn.ampproject.org/v0/amp-dailymotion-0.1.js"></script>
+  <script async custom-element="amp-twitter" src="https://cdn.ampproject.org/v0/amp-twitter-0.1.js"></script>
+  <script async custom-element="amp-facebook" src="https://cdn.ampproject.org/v0/amp-facebook-0.1.js"></script>
+  <script async custom-element="amp-brightcove" src="https://cdn.ampproject.org/v0/amp-brightcove-0.1.js"></script>  
+  <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+  section[expanded] span.expanded {
+    display: block;
+  }
+  section[expanded] span.collapsed {
+    display: none;
+  }
+  section:not([expanded]) span.expanded {
+    display: none;
+  }
+  section:not([expanded]) span.collapsed {
+    display: block;
+  }
+  .blue-box, .red-box, .green-box {
+      width: 100%;
+      height: 280px;
+    }
+    .blue-box {
+      background: blue;
+    }
+    .green-box {
+      background: green;
+    }
+    .red-box {
+      background: red;
+    }
+    amp-carousel {
+      background: black;
+    }
+    .father {
+      display: flex;
+      flex-flow: column;
+      justify-content: center;
+      height: 100%;
+    }
+    .child {
+      width: 100%;
+      height: 55%;
+    }
+
+
+</style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+    <!-- #### amp-video embedded in an amp-carousel component -->
+  <!-- Display a list of videos as slides. For more info about `amp-carousel` take a look [here](https://ampbyexample.com/components/amp-carousel/). -->
```

Comment:
because the type carousel does not support responsive: https://github.com/ampproject/amphtml/blob/master/extensions/amp-carousel/amp-carousel.md, is this correct? Same thing in [here](https://github.com/ampproject/amp-by-example/blob/master/src/30_Advanced/Image_Galleries_with_amp-carousel.html)